ci-operator/step-registry/baremetal/lab/pre/load-balancer/baremetal-lab-pre-load-balancer-commands.sh

```diff
@@ -16,72 +16,78 @@ num_workers="$(yq e '[.[] | select(.name|test("worker"))]|length' "$SHARED_DIR/h
 for bmhost in $(yq e -o=j -I=0 '.[]' "${SHARED_DIR}/hosts.yaml"); do
   # shellcheck disable=SC1090
   . <(echo "$bmhost" | yq e 'to_entries | .[] | (.key + "=\"" + .value + "\"")')
+  # shellcheck disable=SC2154
+  if [ ${#name} -eq 0 ] || [ ${#ip} -eq 0 ] || [ ${#ipv6} -eq 0 ]; then
+    echo "Error when parsing the Bare Metal Host metadata"
+    exit 1
+  fi
+
   if [[ "$name" =~ bootstrap* ]] || [[ "$name" =~ master* ]]; then
     MC="$MC
-      server $name $ip:22623 check inter 1s"
+  server $name $ip:22623 check inter 1s
+  server $name-v6 [$ipv6]:22623 check inter 1s"
     APISRV="$APISRV
-      server $name $ip:6443 check inter 1s"
+  server $name $ip:6443 check inter 1s
+  server $name-v6 [$ipv6]:6443 check inter 1s"
   fi
   if [ "$num_workers" -eq 0 ] || [[ "$name" =~ worker* ]]; then
     INGRESS80="$INGRESS80
-      server $name $ip:80 check inter 1s"
+  server $name $ip:80 check inter 1s
+  server $name-v6 [$ipv6]:80 check inter 1s"
     INGRESS443="$INGRESS443
-      server $name $ip:443 check inter 1s"
+  server $name $ip:443 check inter 1s
+  server $name-v6 [$ipv6]:443 check inter 1s"
   fi
 done
 echo "Generating the template..."
 
 cat > "$SHARED_DIR/haproxy.cfg" <<EOF
 global
-log         127.0.0.1 local2
-pidfile     /var/run/haproxy.pid
-daemon
+  log         127.0.0.1 local2
```

Comment:
Hi @jadhaj can we limit the changes here at the only enablement-related ones?  If any refactoring should be done, it should go in a separate PR.